data/MB/TASSO_34.8_LA_LABAR.yaml

```diff
@@ -0,0 +1,73 @@
+dependent_variables:
+  - header: {title: "TASSO at 34.8 GeV, Lambda Multiplicity", titlepy: "TASSO at 34.8 GeV, Lambda Multiplicity", xlabel: z, xlabelpy: $z$, ylabel: "#frac{s}{#beta}#frac{d#it{#sigma}}{d#it{z}} [pb GeV^{2}]", ylabelpy: "$\\frac{s}{\\beta}\\frac{d\\sigma}{dz}[\\rm{pb GeV^{2}}]$"}
+    qualifiers:
+      - {name: process, value: SIA}
+      - {name: Vs, value: 34.8, units: GeV}
+      - {name: prefactor, value: 1}
+      - {name: z, low: 0.069, high: 1, integrate: false}
+      - {name: hadron, value: LA}
+      - {name: charge, value: 0}
+    values:
+      - errors:
+          - {label: unc, value: minus, -142000: plus, 148000}
```

Comment:
Mont Blanc is not capable of reading asymmetric uncertainties. These must be symmetrised using the D'Agostini prescription (the one that @enocera quoted in his email). Specifically:
```
    semi_diff = (delta_plus + delta_minus) / 2
    average = (delta_plus - delta_minus) / 2
    se_delta = semi_diff
    se_sigma = sqrt(average * average + 2 * semi_diff * semi_diff)
    return se_delta, se_sigma
```
Taking the bin above as an example, `delta_plus` and `delta_minus` are
```
   delta_plus = 148000
   delta_minus = -142000
```
Note that in the [paper](https://arxiv.org/pdf/physics/0403086) D'Agostini takes the absolute value of $\Delta^{\pm}$, and thus the equations above for `semi_diff` and `average` have different signs wrt eqs.(25) and (27).